.github/ISSUE_TEMPLATE/add-new-config.yml

```diff
@@ -506,6 +506,13 @@ body:
       - "false"
   validations:
     required: true  
+- type: textarea
+  id: admin_access
+  attributes:
+    label: Admin Access Emails
+    description: Please enter a comman separated list of emails to which we will grant access to the admin dashboard. This will usually be a relatively short list.
```

Comment:
We used to have a limit of 3 users in the appendix. Maybe we can add a check for this number in the validation and limit it to less than 5?